samples/FakeItEasy.Examples.ruleset

```diff
@@ -15,4 +14,4 @@
     <Rule Id="CA2201" Action="None" />
     <Rule Id="CA2210" Action="None" />
   </Rules>
-</RuleSet>
\ No newline at end of file
+</RuleSet>

```

Comment:
I think this is the same effect that @thomaslevesque mentioned recently somewhere. The editorconfig only applies when VS is used as an editor. I was working in a funny GUI.

You're suggesting touching the file to say that it's okay to have EOLs in rulesets?